src/renderer/components/tooltip/tooltip.tsx

```diff
@@ -64,6 +64,10 @@ export class Tooltip extends React.Component<TooltipProps> {
     autoBind(this);
   }
 
+  @computed get renderToDOM() {
```

Comment:
Looks like a method name. 
Moreover I'd like to keep this more simple. We have already `this.isContentVisible` and `this.isVisible`, isn't it too much for about almost same thing?
So the only thing we should check in render before showing actual DOM is: 

```tsx
render() {
 if(!this.isVisible || (this.isVisible && !this.isContentVisible)) return null;
 // otherwise render content
}
```

Reply:
I have removed this getter and just inlined it.